src/NumericalAlgorithms/Interpolation/CMakeLists.txt

```diff
@@ -27,8 +27,10 @@ add_dependencies(
 
 target_link_libraries(
   ${LIBRARY}
-  INTERFACE ApparentHorizons
-  INTERFACE Domain
-  INTERFACE ErrorHandling
-  INTERFACE Spectral
+  PUBLIC
+  ApparentHorizons
+  Blas
```

Comment:
For my own knowledge, why do many of these need `Blas` to be public? Are the Blas functions used in header files?

Reply:
yes they include `Utilities/Blas.hpp` in headers. I expect some code can be refactored to avoid that, but I haven't attempted that in this PR

Reply:
Sounds good to me!